src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -0,0 +1,596 @@
+                table() //
+                        .row("K1", "L1", "K1", "R1") //
+                        .row("K3", "L3", null, null) //
+                        .row(null, "L2", null, null) //
+                        .row(null, null, "K2", "R2") //
+                        .row(null, null, null, "R3") //
+                        .matches());
+    }
+
+    @Test
+    void CreateVirtualSchemaWithNonexistentConnectionThrowsException() {
+        final String sql = "CREATE VIRTUAL SCHEMA VIRTUAL_SCHEMA_NONEXISTENT_CONNECTION\n" //
+                + "USING " + adapterScript.getFullyQualifiedName() + " WITH\n" //
+                + "SQL_DIALECT = 'EXASOL'\n" //
+                + "CONNECTION_NAME = 'NONEXISTENT_CONNECTION'\n" //
+                + "SCHEMA_NAME = '" + this.sourceSchema.getFullyQualifiedName() + "' ";
+        final SQLException exception = assertThrows(SQLException.class, () -> query(sql));
+        assertThat(exception.getMessage(),
+                containsString("Could not access the connection information of connection \"NONEXISTENT_CONNECTION\""));
+    }
+
+    @Test
+    void testCreateVirtualSchemaWithIgnoreErrorsProperty() throws SQLException {
+        final Table table = createSingleColumnTable("BOOLEAN").insert(true);
+        final Map<String, String> properties = new HashMap<>();
+        properties.putAll(getConnectionSpecificVirtualSchemaProperties());
+        properties.put("IGNORE_ERRORS", EXASOL_TIMESTAMP_WITH_LOCAL_TIME_ZONE_SWITCH);
+        this.virtualSchema = objectFactory.createVirtualSchemaBuilder("VIRTUAL_SCHEMA_IGNORES_ERRORS") //
+                .sourceSchema(this.sourceSchema) //
+                .adapterScript(adapterScript) //
+                .dialectName(EXASOL_DIALECT) //
+                .properties(properties) //
+                .connectionDefinition(this.jdbcConnection) //
+                .build();
+        assertThat(query("SELECT NOW() - INTERVAL '1' MINUTE FROM " + getVirtualTableName(this.virtualSchema, table)),
+                instanceOf(ResultSet.class));
+    }
+
+    @Test
+    void testCreateVirtualSchemaWithoutIgnoreErrorsPropertyThrowsException() throws SQLException {
+        final Table table = createSingleColumnTable("BOOLEAN").insert(true);
+        this.virtualSchema = createVirtualSchema(this.sourceSchema);
+        final SQLException exception = assertThrows(SQLException.class, () -> query(
+                "SELECT NOW() - INTERVAL '1' MINUTE FROM " + getVirtualTableName(this.virtualSchema, table)));
+        assertThat(exception.getMessage(),
+                containsString("Attention! Using literals and constant expressions with datatype "
+                        + "`TIMESTAMP WITH LOCAL TIME ZONE` in Virtual Schemas can produce an incorrect results"));
+    }
+
+    /**
+     * The CastAssertionBuilder is a convenience class that helps formulating cast assertions in a more readable way.
+     */
+    static class CastAssertionBuilder {
```

Comment:
Just a nitpick, I would name this as `CastAssertionVerifier` but it's just a suggestion